pom.xml

```diff
@@ -130,6 +130,14 @@
         <version>${version.org.apache.cassandra}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.wildfly.bom</groupId>
+        <artifactId>jboss-javaee-7.0-wildfly</artifactId>
+        <version>${version.org.wildfly}</version>
```

Comment:
Would it make sense to limit that to the module that defines javax.ejb.* as it looks like no other EE feature is used